django/db/backends/oracle/schema.py

```diff
@@ -250,3 +289,16 @@ def _alter_column_collation_sql(
         return super()._alter_column_collation_sql(
             model, new_field, new_type, new_collation, old_field
         )
+
+    def _alter_column_comment_sql(self, model, new_field, new_type, new_db_comment):
+        return None, [
+            (
+                self.sql_alter_column_comment
+                % {
+                    "table": self.quote_name(model._meta.db_table),
+                    "column": self.quote_name(new_field.column),
+                    "comment": self.quote_comment_value(new_db_comment),
+                },
+                [],
```

Comment:
This (or similar code) is generated in many places in all backends. We should add an internal hook to share this logic, e.g. `BaseDatabaseSchemaEditor._alter_column_comment_sql()`.

Reply:
IMO, `_alter_column_comment_sql()` should return only `sql`, `params`. This would allow it to reused it other places, e.g. in `create_model()` or `add_field()` on Oracle:
```python
def _alter_column_comment_sql(self, model, new_field, new_type, new_db_comment, old_db_comment):
    if (
        not self.connection.features.supports_comments
        or new_field.many_to_many
        or old_db_comment == new_db_comment
    ):
        return "", []
    return (
        self.sql_alter_column_comment
        % {
            "table": self.quote_name(model._meta.db_table),
            "column": self.quote_name(new_field.column),
            "comment": self.quote_comment_value(new_db_comment),
        },
        [],
    )
```